src/helpers/validate.js

```diff
@@ -9,7 +9,7 @@ function validateURL(url) {
 }
 
 function validateFlags(flags) {
-  const mask = /^[\w\.\-]{1,45}$/
+  const mask = /^[\w.-]{1,45}$/
```

Comment:
This might be better to leave it as is because it shows that *those are definitely* the literal characters. An unescaped period might appear to be a wildcard even though it actually matches a literal period.

Reply:
The linter yelled at me for that one. 😆

Reply:
Do you mind suggesting it back? I'm afk on mobile, so can't make the code changes myself right now.